src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = [...table.rows];
+
+rows.forEach((row) => {
+  const cellContent = row.cells[1].innerHTML;
+  const newCells = row.insertCell(row.cells.length - 1);
+  const th = `<th>${cellContent}</th>`;
+  const td = `<td>${cellContent}</td>`;
+  const nameTableSection = row.parentNode.tagName;
+
+  if (nameTableSection === 'THEAD' || nameTableSection === 'TFOOT') {
+    newCells.outerHTML = th;
+  } else {
+    newCells.outerHTML = td;
```

Comment:
Be aware that `outerHTML` replaces the whole element, including the element itself. This could lead to unexpected behavior if you plan to use `newCells` later in your code. If you only want to replace the content of the element, consider using `innerHTML` instead.